src/replica/replica_stub.cpp

```diff
@@ -2299,6 +2299,41 @@ void replica_stub::register_ctrl_command()
                 });
             }));
 
+        _cmds.emplace_back(::dsn::command_manager::instance().register_single_command(
```

Comment:
How about simplify the commands in format of `replica.plog-gc [true|false]` by `register_bool_command()`, like some other remote-commands, e.g. `replica.deny-client`.